package/MDAnalysis/core/universe.py

```diff
@@ -407,6 +407,17 @@ def __init__(self, topology=None, *coordinates, all_coordinates=False,
             self._trajectory.add_transformations(*transformations)
 
         if guess_bonds:
+            warnings.warn(
+                "Bond guessing through the `guess_bonds` keyword is deprecated"
```

Comment:
Add a deprecation doc string under `guess_bonds` with this information so that it shows up in the docs.